lang/src/org/partiql/lang/eval/builtins/SubstringExprFunction.kt

```diff
@@ -25,7 +25,6 @@ import org.partiql.lang.eval.stringValue
 import org.partiql.lang.types.FunctionSignature
 import org.partiql.lang.types.StaticType
 import java.lang.Integer.max
-import java.lang.Integer.min
```

Comment:
`java.lang.Integer.max` is still imported. Does ``java.lang.Integer.min` need removed?

Reply:
Fixed.